app/Repositories/PostRepository.php

```diff
@@ -127,9 +127,11 @@ public function create(array $data, $signupId)
             $signup->save();
         }
 
-        // Edit the image if there is one
-        if (isset($data['file'])) {
-            $this->crop($data, $post->id);
+        if (! config('features.glide')) {
```

Comment:
I know this is not part of the PR, but we might want to remove the `->orientate()` from above too:

```diff
- $image = Image::make($data['file'])->orientate();
+ $image = Image::make($data['file']);
 $fileUrl = $this->aws->storeImage((string) $image->encode('data-url'), $signupId);
```

Reply:
ah yes!

Reply:
@sbsmith86 I'm not seeing this change (remove call to `orientate` on original image)

Reply:
@sbsmith86 hmm yeah I was expecting a change to remove `orientate` here: https://github.com/DoSomething/rogue/pull/634/files#diff-158da9223a5d32ce24f63cb94477a423R69

Reply:
OH! looks like I missed that one. Good 👀 !!!